### ISSUE TITLE
New mode `remote_build_event_upload=none`

### ISSUE THREAD

**Issue by vogelsgesang** [bazelbuild/bazel#19594]
### Description of the feature request:

Currently, I can set `remote_build_event_upload` to either `all` or `minimal`. 
From the documentation:
> If set to 'minimal', local outputs referenced by BEP are not uploaded to the remote cache, except for files that are important to the consumers of BEP (e.g. test logs and timing profile). bytestream:// scheme is always used for the uri of files even if they are missing from remote cache

I would like to have a `remote_build_event_upload=none` mode in which no BEP-related files are uploaded to the remote cache and where the files in the BEP keep are referenced via `file:` instead of `bytestream:` 

### Which category does this issue belong to?

Remote Execution

### What underlying problem are you trying to solve with this feature?

In our use case, we don't want to upload anything BEP-related (test logs, timing profiles, ...) to the remote cache. 
We want to use a remote cache to speed up our builds.
We want to use `--build_event_json_file` to get a local JSON representation of the BEP which we can then analyze using a simple Python script from local disk on the same machine as the original Bazel invocation.

The fact that using a remote cache replaces the `file:` URLs by `bytestream:` URLs gets in our way here

### Which operating system are you running Bazel on?

Linux

### What is the output of `bazel info release`?

release 6.3.2

### Have you found anything relevant by searching the web?

https://www.buildbuddy.io/blog/bazels-remote-caching-and-remote-execution-explained mentions the flag `[--experimental_build_event_upload_strategy=local](https://docs.bazel.build/versions/5.0.0/command-line-reference.html#flag--experimental_build_event_upload_strategy)`,  but I couldn't find more in-depth documentation around it. #10183 also contains some hints that `experimental_build_event_upload_strategy=local` might be what I am looking for?

Maybe this is doing what I want? But then again: Why is there both `experimental_build_event_upload_strategy` and `remote_build_event_upload`? Can / should they be merged into the same setting?

**Comment by brentleyjones** [comment]
You are correct that currently you can use `--experimental_build_event_upload_strategy=local` to get what you want. It existed before the flag that you mentioned, which is why they both exist. I think adjusting the flag for a `none` value could work, and we could possibly remove the older flag.

cc: @coeuvre
